kivy/modules/touchring.py

```diff
@@ -15,6 +15,15 @@
         Scale of the image.
     `alpha`: float, defaults to 1.
         Opacity of the image.
+    `show_cursor`: boolean, default to False
```

Comment:
one quick question to use it in a python script i need to add the code to my python program to be able to see the mouse gestures on my rpi